src/sqlfluff/core/rules/std/__init__.py

```diff
@@ -0,0 +1,20 @@
+# No docstring here as it would appear in the rules docs.
+# Rule definitions for the standard ruleset, dynamically imported from the directory.
+# noqa
+
+import os
+from importlib import import_module
+from glob import glob
+
+rules_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), "L*.py")
+
+for module in sorted(glob(rules_path)):
+    # Manipulate the module path to extract the filename without the .py
+    rule_id = module.split("/")[-1][:-3]
```

Comment:
Will `"/"` work on Windows? This may be more portable (and readable?):
```
os.path.splitext(os.path.basename(module))[0]
```

Reply:
Much better - thanks